bin/node/cli/src/command.rs

```diff
@@ -206,5 +206,12 @@ pub fn run() -> Result<()> {
 		Some(Subcommand::TryRuntime) => Err("TryRuntime wasn't enabled when building the node. \
 				You can enable it with `--features try-runtime`."
 			.into()),
+		Some(Subcommand::BlockchainInfo(cmd)) => {
+			let runner = cli.create_runner(cmd)?;
+			runner.async_run(|config| {
+				let PartialComponents { backend, task_manager, .. } = new_partial(&config)?;
```

Comment:
This is starting a node. It may be possible to just use the DB here and extract the data from that without the node.  
For example like [here](https://github.com/paritytech/substrate/blob/580ebae17fa30082604f1c9720f6f4a1cfe95b50/utils/frame/benchmarking-cli/src/storage/cmd.rs#L108).

Reply:
Thank you for your suggestion. I totally agree, and that was my first take when I started writing this in the Moonbeam repo, but found the `utils` module is private and there are no public re-exports for the `open_database`, `read_meta`.. functions, so I decided to use the service backend.

Now that this is proposed as part of Substrate we can declare the public re-exports and move away from the service.

Reply:
@cheme suggested using the service builder, I think is the cleanest way.